content/04-guides/01-database-workflows/06-cascading-deletes/01-postgresql.mdx

```diff
@@ -532,7 +532,7 @@ const { PrismaClient } = require('@prisma/client');
 const prisma = new PrismaClient();
 
 async function main() {
-  const userWithPost = await prisma.oneMoreUser.create({
+  const userWithPost = await prisma.$oneMoreUser.create({
```

Comment:
Why is there a $ here? 🤔

Reply:
and below too (same thing)

Reply:
Urgh - I searched for `.on. Good catch!

Reply:
Classic search and replace 😄 Happens to everyone once in a while 💥